docs/reference/components/amqp.cue

```diff
@@ -0,0 +1,116 @@
+package metadata
```

Comment:
Nice, thanks! I think this file probably needs a `cue fmt`.

Reply:
This should be good now